src/items/class-features/celestial_aegis_(su).json

```diff
@@ -0,0 +1,115 @@
+{
+  "_id": "AeQXGJzKO43Y7SGM",
+  "name": "Celestial Aegis (Su)",
+  "type": "feat",
+  "img": "systems/sfrpg/images/cup/gameplay/photon.webp",
+  "system": {
+    "type": "",
+    "ability": "",
+    "abilityMods": {
+      "parts": []
+    },
+    "actionTarget": "",
+    "actionType": "",
+    "activation": {
+      "type": "move",
+      "condition": "",
+      "cost": null
+    },
+    "area": {
+      "effect": "",
+      "shape": "",
+      "units": "none",
+      "value": null
+    },
+    "attackBonus": 0,
+    "chatFlavor": "",
+    "critical": {
+      "effect": "",
+      "parts": []
+    },
+    "damage": {
+      "parts": []
+    },
+    "damageNotes": "",
+    "description": {
+      "chat": "",
+      "gmnotes": "",
+      "short": "",
+      "unidentified": "",
+      "value": "<p>&gt;@UUID[Compendium.sfrpg.class-features.G7p2YIGFTAU6uGjL]{Stellar Revelation}</p>\n<p><sub>&nbsp;</sub>@Icon[type:photon]Photon Revelation</p>\n<p>As a move action, you channel the Positive Energy Plane’s power directly into your @UUID[Compendium.sfrpg.class-features.zM7oc1qiNLAPoH1J]{Solar Armor} or @UUID[Compendium.sfrpg.class-features.ylRInRfOfx6nsbUv]{Solar Shield}, helping you capture and redirect your escaping life energy to bolster your body. As a reaction when you take damage that causes you to lose at least 2 Hit Points, you gain temporary Hit Points equal to half the number of Hit Points you lost from the attack or effect. These temporary Hit Points last 3 rounds.</p><p class=\"MsoNormal\">If you’re attuned or fully attuned, you also gain spell resistance equal to 11 + your solarian level against necromancy spells, and against spells with the cold, darkness, or death descriptors.</p>"
```

Comment:
Get rid of `class=\"MsoNormal\"`. How'd that get there? Appears in a few other places